src/App.js

```diff
@@ -39,20 +50,33 @@ class App extends React.Component {
 
   render() {
     return (
-      <>
-        <input onChange={(event) => this.setState({ searchQuery: event.target.value })} placeholder="search for a city"></input>
-        <button onClick={this.getLocation}>Explore!</button>
+      <Container>
+        <Form>
+          <Form.Label>Enter your search..</Form.Label>
+          <Form.Control onChange= {this.handleChange} value={this.state.searchQuery}/>
+          <Button varient="dark" onClick = {this.getLocation}>Explore!</Button>
+
+        </Form>
+        {/* <input onChange={(event) => this.setState({ searchQuery: event.target.value })} placeholder="search for a city"></input>
+        <button onClick={this.getLocation}>Explore!</button> */}
```

Comment:
Looks like you got this changed, so nice work! (I  left a comment in a previous PR, so disregard that :) )